ansible_rulebook/cli.py

```diff
@@ -134,6 +135,11 @@ def get_parser() -> argparse.ArgumentParser:
         "--controller-token",
         help="Controller API authentication token",
     )
+    parser.add_argument(
+        "--print_events",
```

Comment:
To remain consistent with the other args I think this should be `--print-events`.